File: Trapezoid_area/trapezoid.py
# INPUTS
smaller_base = eval(input("Enter the smaller base: "))
larger_base = eval(input("Enter the larger base: "))
height = eval(input("Enter the height: "))

# PROCESS
trapezoid_area = ((smaller_base + larger_base) / 2) * height

# OUTPUT
print("The Area of Trapezoid = {}".format(trapezoid_area))

File: Trapezoid_area/trapezoid.cpp
# include <iostream>
using namespace std;
int main (){

    // INPUTS
    double smaller_base;
    double larger_base;
    double height;

    cout << "Enter the smaller base: ";
    cin >> smaller_base;
    cout << "\nEnter the larger base: ";
    cin >> larger_base;
    cout << "\nEnter the height: ";
    cin >> height;

    // PROCESS
    double area;
    area = ((smaller_base + larger_base) / 2) * height;

    //OUTPUT
    cout << "The area of Trapezoid = " << area << endl;

    return 0;
}